service.go

```diff
@@ -251,7 +277,15 @@ type FrameBuffer struct {
 }
 
 // NewFrameBuffer starts an X virtual frame buffer running in the background.
+//
+// This is equivalent to calling NewFrameBufferWithOptions with an empty map.
 func NewFrameBuffer() (*FrameBuffer, error) {
+	return NewFrameBufferWithOptions(FrameBufferOptions{})
+}
+
+// NewFrameBufferWithOptions starts an X virtual frame buffer running in the background.
+// You may pass in additional options (the "FrameBufferOption*" constants) to change its behavior.
```

Comment:
nit: For at least this package, avoid the second person in technical documentation.

Also, this needs an update for the new function signature.